moveit_ros/moveit_servo/config/panda_simulated_config.yaml

```diff
@@ -47,9 +47,6 @@ robot_link_command_frame:  panda_link0  # commands must be given in the frame of
 
 ## Stopping behaviour
 incoming_command_timeout:  0.1  # Stop servoing if X seconds elapse without a new command
-# If 0, republish commands forever even if the robot is stationary. Otherwise, specify num. to publish.
-# Important because ROS may drop some messages and we need the robot to halt reliably.
-num_outgoing_halt_msgs_to_publish: 4
```

Comment:
Drop this functionality and always just keep republishing, even if the robot is stationary.